components/layout/Footer.tsx

```diff
@@ -143,6 +146,40 @@ function Footer() {
           </Grid>
         </FooterSection>
 
+        <FooterSection title="Find us on socials" sx={{ display: { xs: 0, md: 'block' } }}>
+          <Stack
+            className="social-links"
+            direction="row"
+            spacing={1.5}
+            sx={{ justifyContent: { xs: 'center', md: 'flex-start' } }}
```

Comment:
I think that we need to center on small instead of xs, since small is the breakpoint down for mobile. It looks ever so slightly to the right at the moment.

Reply:
Updated. set to center for small.